sfs/mono/__init__.py

```diff
@@ -3,14 +3,23 @@
 .. autosummary::
     :toctree:
 
-    drivingfunction
+    esaedge
+    nfchoa
+    sdm
+    wfs
```

Comment:
This is alphabetic?

Shouldn't it be somehow "by importance"?

Or probably chronologically with respect to the SFS-Toolbox development?

Reply:
Alphabetic is the easiest to understand and you don't have to fight about which is more important.

Reply:
alphabetic was indeed intentional. I don't have a preference.